resources_pages/install_ds_stack_mac.md

```diff
@@ -665,15 +665,40 @@ HISTFILESIZE=50000
 # (e.g. opening the terminal and typing or pasting commands).
 # If you run a bash script or makefile,
 # the original command is used instead of your alias.
-# More verbose file list format
-alias l='ls -vlrthAF'
+# ---
+# MDS help script
+alias mds-help='bash ~/.mds-help.sh'
+# Some common operations
+alias l='ls -lthAF'
+alias jl='jupyter lab'
+alias ca='conda activate'
+alias gl='git log --date short -10 --pretty=format:"%C(auto,yellow)%h %C(auto,blue)%ad%C(auto)%d %C(auto,reset)%s"'
+alias gt='git status'
+alias gm='git commit -m'
+alias gap='git add -p'
 # Move, remove, and copy show what was done and prompt before overwriting files.
 alias rm="rm -vI"
 alias mv="mv -vi"
 alias cp="cp -vi"
 alias mkdir="mkdir -vp"
+# Case insensitive grep by default
+alias grep='grep -i'
 ```
 
+Finally, download and save the MDS help script via the following command.
+
+```
+curl -Sso ~/.mds-help.sh https://raw.githubusercontent.com/UBC-MDS/UBC-MDS.github.io/master/resources_pages/mds-help.sh
```

Comment:
I think it would be beneficial to remind the students to run this line in their home directory. Otherwise the shell file `mds-help.sh` might unintentionally end up somewhere else, causing `alias mds-help='bash ~/.mds-help.sh'` to fail.